README.adoc

```diff
@@ -1,4 +1,4 @@
-= Red Hat CodeReady Containers - OpenShift 4 on your Laptop
+= Container Runtime Control - OpenShift 4 and Podman on your Laptop
```

Comment:
The new naming feels contrived and artificial to me :) Why not just call it `CRC` without having an official long name? 
`Running OpenShift 4 and Podman containers on your Laptop` reads better if it's not too long.

Reply:
To me `Containter Runtime Control` is better than `CodeReady Containers` but what are we controlling here since we only ship single container runtime for both offering (openshift/podman). If we can make `crc` as it without any abbreviation then it would be great.

Reply:
We settled on referring to `crc` and the ability to `crc runs containers`. It will not have any direct meaning to the name.

Note: the CodeReady branding is to be removed and can not be referred to anymore.